### PR TITLE
Task/update deps

### PR DESCRIPTION
**Pull request by JFriel**
Update the dependencies to fix security issues and general bump of versions used

### CODE REVIEW

File: HICPlugin/HICPlugin.csproj [hunk 1]
```diff
@@ -34,6 +34,9 @@
   <ItemGroup>
     <None Include="Resources\MTo1.png" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Drawing.Common" Version="8.0.6" />
```
Comment (jas88) [inline HICPlugin/HICPlugin.csproj:38]: Where did this one sneak in from...?

File: RDMP/Directory.Packages.props [hunk 2]
```diff
@@ -3,26 +3,26 @@
 		<PackageVersion Include="CommandLineParser" Version="2.9.1"/>
```
Comment (jas88) [inline RDMP/Directory.Packages.props:3]: Er, this file isn't even part of HICplugin...

File: HICPlugin/HICPlugin.csproj (hunk 1 above)
Comment (JFriel) [reply HICPlugin/HICPlugin.csproj:38]: Looks like it's needed with the move to net8  https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only  not sure show this was working up till now...

Comment (jas88) [reply HICPlugin/HICPlugin.csproj:38]: If it's needed, it will be needed within RDMP.UI not within HICPLUGIN

**Review by jas88 — APPROVED** [review]

**Pull request merged**